app/web/src/store/components.store.ts

```diff
@@ -1093,6 +1149,7 @@ export const useComponentsStore = (forceChangeSetId?: ChangeSetId) => {
               idsToDelete.forEach((id) => {
                 delete this.rawEdgesById[id];
               });
+              // TODO do this for subscriptions
```

Comment:
We should probably heed this ... I think it's probably easier now that we finished the type changes!

Reply:
Gonna leave this one for the next PR